javascripts/content_scripts/privly.js

```diff
@@ -390,6 +390,9 @@ var privly = {
     //Set the source URL
     iFrame.setAttribute("src", applicationUrl);
 
+    //URL which it will be replacing
+    iFrame.setAttribute("data-privlyHref",object.getAttribute("data-privlyHref"));
```

Comment:
@smcgregor Yes. It is set for the element 'a'. And, 'a' is local to that method. I just have access to iframe element here, so had to put the original link as a new attribute into iframe. 
Is there a way I could get 'a' corresponding to the iframe directly?